src/webview/js/js.js

```diff
@@ -40,31 +42,16 @@ import type { MessageListEvent } from '../webViewEventHandlers';
 /** Like RN's `Platform.OS`. */
 const platformOS = window.navigator.userAgent.match(/iPhone|iPad|iPod/) ? 'ios' : 'android';
 
-/**
- * Convert an array-like object to an actual array.
- *
- * A substitute for `Array.from`, which doesn't exist in the WebViews of some
- * of our supported platforms.
- */
-function arrayFrom<T>(arrayLike: HTMLCollection<T>): T[] {
-  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#Array-like_objects
-  return Array.prototype.slice.call(arrayLike);
-}
+/* eslint-disable no-extend-native */
 
-/**
- * A substitute for `Array.from`, for `NodeList` input.
- *
- * See `arrayFrom`.  Duplicated to satisfy Flow.
- */
-// On at least Flow v0.78.0, writing `HTMLCollection<T> | NodeList<T>`
-// worked fine for `HTMLCollection` arguments, but not `NodeList`.  Shrug.
-function arrayFromNodes<T>(arrayLike: NodeList<T>): T[] {
-  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#Array-like_objects
-  return Array.prototype.slice.call(arrayLike);
+/** Polyfill Array.from. Native in Chrome 45 and at least Safari 13. */
+if (!Array.from) {
+  // $FlowFixMe (polyfill)
+  Array.from = function from(arr: Iterable<T>): Array<T> {
+    return Array.prototype.slice.call(arr);
+  };
```

Comment:
One slightly awkward consequence of this strategy is that if you hover on `Array.from` at a call site, you'll see Flow has a quite complicated type for it -- supports a couple more arguments and several ways to call them. Presumably those reflect the real semantics of `Array.from` in the spec and in current browsers.

So if we write `Array.from` at call sites, we could end up using those fancy features without realizing there was a problem; it'd pass Flow, and it'd work in newer browsers, but then fail in browsers that used the polyfill.

One approach to that would be to make the polyfill more complete. But that sounds like a rabbit hole, and I definitely wouldn't be excited about pursuing that solution.

What's the motivation for converting from the existing functions here? If it's to make them available from a separate module, perhaps we should just pull these out first to a "backports" module that both can import from.

Reply:
No, that was basically it. I don't think this actually increases our risk profile, though: right now we don't have any way to keep people from calling `Array.from` in this code _at all_.

Making the polyfill more complete is ... probably not an issue? The additional features (that Flow knows about) fall into two categories:
* Use of the second parameter. This is for most purposes identical to a following call to `.map()`, and so would be trivial to implement.
* Use with generic `@@iterator`s. If we need support for _that_, we'll have to use `core-js` to polyfill a lot of things, not just `Array.from`!

Reply:
We looked together at [the MDN doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) for this function, and now that I understand what those other parameters do I think the risk is low that anyone working in this code will want to use them. So I'm happy just leaving them out.

Should have a comment saying there are features this polyfill leaves out, though.